lib/detectors/FlashLoan/FlashloanDetector.h

```diff
@@ -72,6 +84,12 @@ namespace vanguard {
             for (auto blk: fn->blocks()) {
                 for (auto ins: blk->instructions()) {
                     ins->accept(*resolver);
+                    CallTargetResolver callResolver;
+
+                    ins->accept(callResolver);
+                    for(auto tgt : callResolver.tgts) {
```

Comment:
nit: can improve maintainability by annotating auto with `*` for pointer types
```suggestion
                    for(auto *tgt : callResolver.tgts) {
```